### PR TITLE
Update price.js

### PR DESCRIPTION
**Pull request by ykris45**
added malaysian ringgit , @filipnyquist please review my friends

### CODE REVIEW

File: bot/modules/price.js [hunk 1]
```diff
@@ -171,6 +176,10 @@ exports.price = {
           url: 'https://blockchain.info/ticker',
           path: '$.CHF.buy'
         },
+        BTCMYR: {
+          url: 'https://api.coinmarketcap.com/v1/ticker/bitcoin/?convert=MYR',
+          path: '$.MYR.buy'
```
[the remark this answers is not in the document]
Comment (ykris45) [reply bot/modules/price.js]: @filipnyquist https://min-api.cryptocompare.com/data/price?fsym=BTC&tsyms=MYR
please review my comment

[the remark this answers is not in the document]
Comment (ykris45) [reply bot/modules/price.js]: image

**Comment by tzarebczan** [comment]
@filipnyquist is this good to go now?

**Comment by filipnyquist** [comment]
I'll take a closer look now

**Comment by ykris45** [comment]
@tzarebczan and @filipnyquist thanks you so much for merge my PR

**Pull request merged**